app/src/routes/mainpage/mainpageCtrl.js

```diff
@@ -0,0 +1,19 @@
+import MainpageService from "../../models/mainpage/mainpageService.js";
+
+export default {
+  getUser: async (req, res) => {
+    const instance = new MainpageService(req);
```

Comment:
```suggestion
    const mainpageService = new MainpageService(req);
```
어떤 클래스의 인스턴스인지 더 명확하게 하는게 낫지 않을까요? 만약 한 메서드에서 클래스 두개를 인스턴스로 만들면 instance, instance2 이렇게 작명하진 않자나용??

Reply:
명확하게 작명하도록 하겠습니다!!